aw_watcher_window/main.py

```diff
@@ -65,9 +67,12 @@ def heartbeat_loop(client, bucket_id, poll_time, exclude_title=False):
         try:
             current_window = get_current_window()
             logger.debug(current_window)
+        except Xlib.error.XError as e:
```

Comment:
Please don't catch Xlib.error errors here. The main.py file is used by mac and windows aswell which do not have the xlib dependency, so please try to catch the exceptions somewhere in the xlib.py file instead.